### PR TITLE
Profiles

### PR DESCRIPTION
**Pull request by artdeell**
Not ready to merge!

### CODE REVIEW

**Comment by 35555555555** [comment]
4444

**Comment by Toothyy** [comment]
Thanks for doing this yes

**Comment by Zeyrox1090** [comment]
lol

File: app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/PojavLoginActivity.java [hunk 1]
```diff
@@ -351,51 +351,52 @@ private void unpackComponent(AssetManager am, String component) throws IOExcepti
...
-        File forgeSplashFile = new File(Tools.DIR_GAME_NEW, "config/splash.properties");
+    public static void disableSplash(String gameFolder) {
+        mkdirs(gameFolder + "/config");
+        File forgeSplashFile = new File(gameFolder, "config/splash.properties");
         String forgeSplashContent = "enabled=true";
         try {
             if (forgeSplashFile.exists()) {
                 forgeSplashContent = Tools.read(forgeSplashFile.getAbsolutePath());
             }
             if (forgeSplashContent.contains("enabled=true")) {
```
Comment (aimiebell) [inline app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/PojavLoginActivity.java:369]: I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [QBA: Method assigns boolean literal in boolean expression (QBA_QUESTIONABLE_BOOLEAN_ASSIGNMENT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#qba-method-assigns-boolean-literal-in-boolean-expression-qba-questionable-boolean-assignment).
This method assigns a literal boolean value (true or false) to a boolean variable inside an if or while expression. Most probably this was supposed to be a boolean comparison using ==, not an assignment using =.

Comment (khanhduytran0) [reply app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/PojavLoginActivity.java:369]: But it was a boolean inside string, that was for assignment a key inside a config file.

**Comment by Harsh-bot-beep** [comment]
Intersting

**Comment by Hidenoud** [comment]
yes

**Comment by Mathias-Boulay** [comment]
Given the electric boogaloo version got merged, this one is useless

**Pull request closed without merging**